src/layout/execute.cpp

```diff
@@ -37,28 +37,7 @@ int execute(chopper::configuration & config,
             std::vector<std::vector<std::string>> const & filenames,
             std::vector<seqan::hibf::sketch::hyperloglog> const & sketches)
 {
-    assert(config.hibf_config.number_of_user_bins > 0);
-
-    if (config.hibf_config.disable_estimate_union)
-        config.hibf_config.disable_rearrangement = true;
-
-    if (config.hibf_config.tmax == 0) // no tmax was set by the user on the command line
-    {
-        // Set default as sqrt(#samples). Experiments showed that this is a reasonable default.
-        if (size_t number_samples = config.hibf_config.number_of_user_bins;
-            number_samples >= 1ULL << 32) // sqrt is bigger than uint16_t
```

Comment:
this is not part of `config.hibf_config.validate_and_set_defaults();`

we should add this before deleting it

Reply:
We use size_t in the library